### ISSUE TITLE
Render HTML in report

### ISSUE THREAD

**Issue by mbhall88** [snakemake/snakemake#478]
Is it possible to render HTML in reports?  

As an example, I currently have a script which collates information for each of my 200 samples based on the results/output of a few different rules and outputs a HTML file which has a table of all of this information. It would be nice if I could have this HTML file rendered in the snakemake report, rather than as an attachment file. I know there is the boilerplate RST files, but they don't allow for performing the kind of logic I need, i.e. reading and performing computation of the contents of multiple files.

If there is a better way of achieving this, I am open to suggestions.

**Comment by KuechlerO** [comment]
Yes, this is possible. Just mark the HTML file of choice with the `report`-flag.
Here's the documentation, but yeah... HTML is not clearly mentionedd in there... But for this worked :)

[Report-Documentation](https://snakemake.readthedocs.io/en/stable/snakefiles/reporting.html#rendering-reports)
